Description: Various changes to the upstream makefiles

* Don't strip binaries in Makefile.
* Install in /usr/games, not /usr/bin.
* Install cheats.xml into /usr/share/games/bsnes/, not ~/.bsnes.
* Attach profile name to binary.

Origin: http://anonscm.debian.org/gitweb/?p=pkg-games/bsnes.git;a=tree;f=debian/patches

--- a/bsnes/Makefile
+++ b/bsnes/Makefile
@@ -25,7 +25,7 @@
 
 # platform
 ifeq ($(platform),x)
-  link += -s -ldl -lX11 -lXext
+  link += -ldl -lX11 -lXext
 else ifeq ($(platform),osx)
 else ifeq ($(platform),win)
   link += -mwindows
--- a/bsnes/ui/Makefile
+++ b/bsnes/ui/Makefile
@@ -87,18 +87,16 @@
 	test -d ../bsnes.app || mkdir -p ../bsnes.app/Contents/MacOS
 	$(strip $(cpp) -o ../bsnes.app/Contents/MacOS/bsnes $(objects) $(link))
 else
-	$(strip $(cpp) -o out/bsnes $(objects) $(link))
+	$(strip $(cpp) -o out/bsnes-$(profile) $(objects) $(link))
 endif
 
 install:
 ifeq ($(platform),x)
-	install -D -m 755 out/bsnes $(DESTDIR)$(prefix)/bin/bsnes
+	install -D -m 755 out/bsnes-$(profile) $(DESTDIR)$(prefix)/games/bsnes-$(profile)
 endif
 	install -D -m 644 data/bsnes.png $(DESTDIR)$(prefix)/share/pixmaps/bsnes.png
 	install -D -m 644 data/bsnes.desktop $(DESTDIR)$(prefix)/share/applications/bsnes.desktop
-	mkdir -p ~/.config/bsnes
-	cp data/cheats.xml ~/.config/bsnes/cheats.xml
-	chmod 777 ~/.config/bsnes ~/.config/bsnes/cheats.xml
+	install -D -m 644 data/cheats.xml $(DESTDIR)$(prefix)/share/games/bsnes/cheats.xml
 
 uninstall:
 ifeq ($(platform),x)
